storage/blockdevice/COMPONENT_QSPIF/mbed_lib.json

```diff
@@ -18,11 +18,13 @@
         "QSPI_POLARITY_MODE": 0,
         "QSPI_FREQ": "40000000",
         "QSPI_MIN_READ_SIZE": "1",
-        "QSPI_MIN_PROG_SIZE": "1"
+        "QSPI_MIN_PROG_SIZE": "1",
+        "QSPI_NUM_STATUS_REGISTER":"2"
     },
```

Comment:
We should really have a "help" section here.  Like:
```json
"QSPI_NUM_STATUS_REGISTER": {
    "help": "For Macronix QSPI flashes, this must be used to set the number of status registers present on the flash.  Ignored for non-Macronix devices."
}
```

Reply:
that would be nice to have. but ouf of scope of  this PR.